File: backend/app/RedisKeys.py
class RedisKeys:
    TOTAL_NR_PIXELS = 'total_nr_pixels'
    BITS_PER_COLOR = 'bits_per_color'
    NR_SEC_COLS = 'sections_nr_cols'
    NR_SEC_ROWS = 'sections_nr_rows'
    SEC_WIDTH = 'section_width'
    SEC_HEIGHT = 'sections_height'
    COLOR_SET = 'color_set'
    SEC_IDS = 'sec_ids'

    @staticmethod
    def sec_info(id):
        return f'sec_{id}'
    
    @staticmethod
    def sec_pixel_data(id):
        return id

File: backend/app/rediskeys.go
package main

import "fmt"

var REDIS_KEYS = struct {
	TOTAL_NR_PIXELS string
	BITS_PER_COLOR  string
	NR_SEC_COLS     string
	NR_SEC_ROWS     string
	SEC_WIDTH       string
	SEC_HEIGHT      string
	COLOR_SET       string
	SEC_IDS         string
	SEC_META        func(string) string
	SEC_PIX_DATA    func(string) string
}{
	"total_nr_pixels",
	"bits_per_color",
	"sections_nr_cols",
	"sections_nr_rows",
	"section_width",
	"sections_height",
	"color_set",
	"sec_ids",
	func(id string) string {
		return fmt.Sprint("sec_", id)
	},
	func(id string) string {
		return id
	},
}
